HW8(MVC)/scripts/View.js

```diff
@@ -0,0 +1,131 @@
+import { $C } from './Controller.js';
+
+class View {
+  constructor() {
+    this.toDoContainer = document.querySelector('.to-do');
```

Comment:
it is better to use id if you are processing only one element.

it is also worth considering error handling if the element is not found